ImageManager/ImageManager.py

```diff
@@ -0,0 +1,89 @@
+"""
+    Handles CRUD operations for images, and their configuration on the
+    FIWARE backend
+"""
+
+import json
+import logging
+from time import time
+from flask import request
+from flask import make_response
+from flask import Blueprint
+from utils import *
+
+from DatabaseModels import *
+from SerializationModels import *
+from TenancyManager import init_tenant_context
+
+from app import app
+
+image = Blueprint('image', __name__)
+
+LOGGER = logging.getLogger('image-manager.' + __name__)
+LOGGER.addHandler(logging.StreamHandler())
+LOGGER.setLevel(logging.INFO)
+
+global_imageid = 20
+
+
+@image.route('/image/<imageid>', methods=['GET'])
+def get_image(imageid):
+    try:
+        print(request)
+        init_tenant_context(request, db)
+        orm_image = assert_image_exists(imageid)
+        result = {
```

Comment:
why not have this include creation/update dates as well?
and if doing so, why not have marshmallow do this for us?